app/rest/proxy/avatar_store_test.go

```diff
@@ -133,3 +145,34 @@ func TestAvatarStore_resize(t *testing.T) {
 		assert.Equalf(t, c.hr, bounds.Dy(), "file %s", c.file)
 	}
 }
+
+func TestAvatarStore_ID(t *testing.T) {
+	p := NewFSAvatarStore("/tmp/avatars.test", 300)
+	err := os.MkdirAll("/tmp/avatars.test/30", 0700)
+	require.NoError(t, err)
+	defer os.RemoveAll("/tmp/avatars.test")
+
+	// file not exists
+	id := p.ID("some_random_name.image")
+	assert.Equal(t, "a008de0a2ccb3308b5d99ffff66436e15538f701", id) // store.EncodeID("some_random_name.image")
+	// file exists
+	err = ioutil.WriteFile("/tmp/avatars.test/30/b3daa77b4c04a9551b8781d03191fe098f325e67.image", []byte("something"), 0666)
```

Comment:
G303: File creation in shared tmp directory without using ioutil.Tempfile

Reply:
@umputun should I do something with this warns?

Reply:
nah, this is bs, especially for tests.